src/test/java/com/google/sps/TextAnalyserTest.java

```diff
@@ -0,0 +1,132 @@
+package com.google.sps.data;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.junit.Test;
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Set;
+
+@RunWith(JUnit4.class)
+public final class TextAnalyserTest {
+
+  @Test
+  public void analysePositiveSentiment() {
+    try {
+      TextAnalyser textAnalyser = new TextAnalyser("Today is a special day. It is your birthday, so enjoy it!");
+      assertTrue("Sentiment score is not positive.", textAnalyser.getSentimentScore() > 0.0F);
+    } catch (IOException e) {
+      fail(e.getMessage());
+    }
+  }
+
+  @Test
+  public void analyseNegativeSentiment() {
+    try {
+      TextAnalyser textAnalyser = new TextAnalyser("I don't like this.");
+      assertTrue("Sentiment score is not negative.", textAnalyser.getSentimentScore() < 0.0F);
+    } catch (IOException e) {
+      fail(e.getMessage());
+    }
+  }
+
+  @Test
+  public void analysePositiveMood() throws IOException {
+    TextAnalyser textAnalyser = new TextAnalyser("Happy Birthday!");
+    String expected = "thrilled";
```

Comment:
`assertEquals("thrilled", textAnalyser.getMood());` is probably more succinct, and totally sufficient─up to you.